pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -121,6 +121,10 @@ func Pod(source *v1alpha1.CatalogSource, name string, image string, labels map[s
 						InitialDelaySeconds: livenessDelay,
 					},
 					Resources: v1.ResourceRequirements{
+						Limits: v1.ResourceList{
+							v1.ResourceCPU:    resource.MustParse("100m"),
```

Comment:
I think this is fair.

FYI - this does not control the creation of any of the pods for the default catalogs, those pods are generated by marketplace.

Reply:
Yep, that is true. It's actually why I thought doing this exercise was a good idea in the first place, since we had to put arbitrary resource limits there before:

https://github.com/operator-framework/operator-marketplace/blob/master/pkg/registry/registry.go#L410

I was hoping to move away from that. I was thinking that once this merges we should maybe update that to match the resource limits described here. We could possibly even consider it part of the same bugzilla?

Reply:
That works for me, I'm going to be in the marketplace codebase today adding a `NodeSelector` to prevent catalogs from being scheduld onto Windows nodes, so I can add the limits here as well.